systemtests/src/test/java/org/bf2/admin/kafka/systemtest/plain/RestEndpointTestIT.java

```diff
@@ -697,4 +697,14 @@ void testOptionsRequestIncludesTwoHourMaxAge() {
                         equalTo(String.valueOf(Duration.ofHours(2).toSeconds())));
     }
 
-}
\ No newline at end of file
+    @Test
+    void testUnknownEndpointReturnsNotFound() {
+        given()
+            .log().ifValidationFails()
+            .when()
+                .get("/foo/bar")
+            .then()
+                .log().ifValidationFails()
+                .statusCode(Status.NOT_FOUND.getStatusCode());
```

Comment:
Better to test not just the status but also that the body matches what is expected for the new `ErrorType`.
```suggestion
        final ErrorType expectedError = ErrorType.RESOURCE_NOT_FOUND;

        given()
            .log().ifValidationFails()
            .when()
                .get("/foo/bar")
            .then()
                .log().ifValidationFails()
            .assertThat()
                .statusCode(expectedError.getHttpStatus().getStatusCode())
                .body("", matchesError(expectedError));
```